packages/web/src/interfaces/social-profile-provider.ts

```diff
@@ -0,0 +1,4 @@
+export enum SocialProfileProvider {
+  FACEBOOK = 'Facebook',
+  GOOGLE = 'Google'
+}
```

Comment:
enums belong in contants, not interfaces, if you're trying to keep everything well-organized